apps/payments/next/.storybook/main.ts

```diff
@@ -0,0 +1,30 @@
+import type { StorybookConfig } from '@storybook/nextjs';
+
+const config: StorybookConfig = {
+  stories: ['../components/**/*.stories.@(js|jsx|ts|tsx|mdx)'],
+  addons: [
```

Comment:
not entirely relevant, but the Monitor team has been having good success w/ [`"@storybook/addon-a11y"`](https://www.npmjs.com/package/@storybook/addon-a11y)

Reply:
We saw that! I remember you mentioning it last year. Lisa mentioned maybe slipping it into this PR. Since SP3 is pretty much all new and experimentation, and this PR rolled into the next sprint, I would like to do this. I'll ping you when it's in. Would love feedback and/or support where necessary!